python/ray/serve/_private/deployment_function_executor_node.py

```diff
@@ -66,16 +70,22 @@ def to_json(self) -> Dict[str, Any]:
             "deployment_function_handle": self._deployment_function_handle,
             "args": self.get_args(),
             "kwargs": self.get_kwargs(),
+            "other_args_to_resolve": self.get_other_args_to_resolve(),
             "uuid": self.get_stable_uuid(),
         }
 
+    def get_return_type(self) -> str:
+        if "return_type_annotation" in self._bound_other_args_to_resolve:
```

Comment:
This pattern is starting to become a bit unwieldy. `_bound_other_args_to_resolve` stores a lot of arbitrary KV pairs that are defined all over the codebase and aren't documented in a centralized location. I think at some point, we should either convert all these to a central `enum` or document them somewhere.

For now, can we add a docstring mentioning that the types are calculated using `type_to_string` from `gradio_utils.py`?